### DISCUSSION TITLE
Keeping track of defined elements

### DISCUSSION THREAD

**Discussion by lem-onade** [digita-ai/semcom#99] [Archived] [answered]
See `RegisterComponentService`

`this.registered` is only useful when the service is never stopped. When the service is reinvoked, `this.registered` is empty, but the element are still defined in `document`.

Perhaps replace with [`customElements`](https://developer.mozilla.org/en-US/docs/Web/API/CustomElementRegistry)

**Comment by woutermont** [comment, marked as answer]
Solved in #127
